### PR TITLE
[FEATURE] HER-5119 change laravel generator admin

### PR DESCRIPTION
**Pull request by tiago-wd**
Change in the package to generate the admin pages.

### CODE REVIEW

File: src/Common/GeneratorConfig.php [hunk 1]
```diff
@@ -205,7 +205,7 @@ public function loadPaths()
         $this->pathViews = config(
             'infyom.laravel_generator.path.views',
             base_path('resources/views/')
-        ).$viewPrefix.$this->mCamelPlural.'/';
+        ).$this->mSnakePlural.'/';
```
Comment (cyril-sf) [inline src/Common/GeneratorConfig.php:208]: @tiago-wd why this change?

**Review by cyril-sf — COMMENTED** [review]
@tiago-wd can you describe why you needed to change the code in this library?

File: src/Common/GeneratorConfig.php (hunk 1 above)
Comment (ericksrdev) [reply src/Common/GeneratorConfig.php:208]: It's changing the view name generation from camel case to snake case

Comment (ericksrdev) [reply src/Common/GeneratorConfig.php:208]: It will make our views to be named like: tenant_industries.blade.php instead of tenantIndustries.blade.php

**Review by ericksrdev — APPROVED** [review]
